assets/training/model_management/tests/unittests/test_downloader.py

```diff
@@ -155,3 +156,31 @@ def test_download_model_with_azureblob_source(self):
                 },
             )
             mock_downloader.download_model.assert_called_once()
+
+    def test_fixed_llama_models_list(self):
+        """Test llama model list."""
+        list_of_models = LlamaModels.list_values()
+
+        allowed_llama_models = ["meta-llama/Llama-2-7b-chat",
+                                "meta-llama/Llama-2-13b-chat",
+                                "meta-llama/Llama-2-70b-chat",
+                                "meta-llama/Llama-2-7b",
+                                "meta-llama/Llama-2-13b",
+                                "meta-llama/Llama-2-70b"
+                                ]
+        for model in list_of_models:
+            self.assertIn(model, allowed_llama_models)
+
+    def test_fixed_llama_hf_models_list(self):
+        """Test llama hf model list."""
+        list_of_models = LlamaHFModels.list_values()
+
+        allowed_llama_hf_models = ["meta-llama/Llama-2-7b-chat-hf",
```

Comment:
Should we test, that for meta huggingface ids we are indeed able to find corresponding id in azureml-meta and call to mlclient.models.get is made (via mocking/patching)?

Reply:
already tested this https://ml.azure.com/runs/5f6ac75d-70a2-4117-90a0-38f535f2bda4?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/model-mgmt-australiaeast&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#